src/rust/engine/process_execution/src/remote.rs

```diff
@@ -1417,6 +1419,16 @@ pub async fn check_action_cache(
       context
         .workunit_store
         .increment_counter(Metric::RemoteCacheRequestsCached, 1);
```

Comment:
This metric increment should probably go after the successful eager_fetch, because we'll treat a failed eager_fetch as a failed cache lookup.